api/people.js

```diff
@@ -32,6 +40,7 @@ async function update(personId, personParams) {
 }
 
 // filter example: {{ops_guide: true, rgl: true}}
+// TODO update with SWR hook
```

Comment:
Do we need to pass in a config into this function?

Reply:
Yes! Adjusted